test/e2e_env/universal/observability/applications_metrics.go

```diff
@@ -206,6 +206,7 @@ metrics:
 		Eventually(func(g Gomega) {
 			stdout, _, err := client.CollectResponse(
 				universal.Cluster, "test-server-dp-metrics", "http://"+net.JoinHostPort(ip, "5555")+"/stats?filter=concurrency",
+				client.WithHeader("Accept", "text/plain; version=0.0.4; charset=utf-8"),
```

Comment:
What happens if we don't set `Accept`?

Reply:
If we don't set `Accept` explicitly, the client requests with `Accept: */*`. In this case the hijacker would set the content-type returned by the applications if that is valid, else as the Accept is `*/*`, it would set the highest priority content-type available in the priority map.

If the `Accept: VALID_PROM_CONTENT_TYPES` and the applications returned invalid content-type, then the hijacker would set the highest priority content type from the Accept header.

To test this I set the Accept to text/plain in one case and openmetrics-text in another, to check if the hijacker honours them.

I would like to test how the hijacker would behave when applications return different valid/invalid content-types. But I am not sure where can I set content-type of applications in this testing framework.